demos/guide/quic-client-non-block.c

```diff
@@ -271,7 +271,7 @@ int main(int argc, char *argv[])
      * verification fails. Virtually all clients should do this unless you
      * really know what you are doing.
      */
-    SSL_CTX_set_verify(ctx, SSL_VERIFY_PEER, NULL);
+    SSL_CTX_set_verify(ctx, SSL_VERIFY_NONE, NULL);
```

Comment:
Why this change??

Reply:
This just got sucked in unintentionally, thanks for catching that. The change got reverted will push shortly.